mw-webapp/src/component/usersBlock/UsersBlock.tsx

```diff
@@ -17,7 +17,8 @@ export const UsersBlock = () => {
   }, []);
 
   const renderUsers = () => {
-    return users.map((user) => (
+    const usersObjectToArray = Object.values(users);
```

Comment:
it is not a converter. It is `usersList`

Reply:
Done